src/hugo/content/en-us/patterns/MVC/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Model-View-Controller Device Software Design"
+weight: 10
+aliases:
+  - /designs/MVC/
+summary: "recommendation for the design and structure of device software"
```

Comment:
Minor change: capitalize "recommendation" to be consistent with the Patterns summary page.